contributivity_measures.py

```diff
@@ -53,3 +55,20 @@ def compute_SV(node_list):
     # Return SV of each node
     return list_shapley_value
 
+
+# Compute independent performance scores of models trained independently on each node
+    
+def compute_independent_scores(node_list, target_score):
+    
+    # Initialize a list of performance scores
+    performance_scores = []
+    
+    # Train models independently on each node and append perf. score to list of perf. scores
+    for node in node_list:
+        performance_scores.append(fl_train_eval.single_train_score(node)[1])
+        
+    # Compute 'regularized' values of performance scores so that they are additive and their sum amount to the target performance score
+    additive_scores = softmax(performance_scores) * target_score
```

Comment:
Yes it can. Well, this is ours to decide, if we think it somehow makes sense for comparison purposes. I think it does. What I am not sure of is what is the *more interesting* for comparison purposes:
- scores of models independently trained vs. score of model trained on all nodes
- additive values computed from these scores vs. exact Shapley values
Both might be good to observe maybe.